tests/Integration/Behaviour/behat.yml

```diff
@@ -217,6 +217,38 @@ default:
                 - Tests\Integration\Behaviour\Features\Transform\CurrencyTransformContext
                 - Tests\Integration\Behaviour\Features\Transform\SharedStorageTransformContext
                 - Tests\Integration\Behaviour\Features\Transform\StringToBoolTransformContext
+        catalog_price_rule:
+          paths:
+            - "%paths.base%/Features/Scenario/CatalogPriceRule"
+          contexts:
+            - Tests\Integration\Behaviour\Features\Context\CommonFeatureContext
+            - Tests\Integration\Behaviour\Features\Context\ShopFeatureContext
+            - Tests\Integration\Behaviour\Features\Context\CurrencyFeatureContext
+            - Tests\Integration\Behaviour\Features\Context\CategoryFeatureContext
+            - Tests\Integration\Behaviour\Features\Context\Domain\CatalogPriceRuleContext
+            - Tests\Integration\Behaviour\Features\Context\Domain\Product\UpdateCategoriesFeatureContext
+            - Tests\Integration\Behaviour\Features\Context\Domain\CategoryFeatureContext
+            - Tests\Integration\Behaviour\Features\Context\Domain\SupplierFeatureContext
+            - Tests\Integration\Behaviour\Features\Context\Domain\Product\UpdateProductSuppliersFeatureContext
+            - Tests\Integration\Behaviour\Features\Context\CatalogPriceRuleContext
+            - Tests\Integration\Behaviour\Features\Context\Domain\ManufacturerFeatureContext
+            - Tests\Integration\Behaviour\Features\Context\Domain\Product\UpdateBasicInformationFeatureContext
+            - Tests\Integration\Behaviour\Features\Context\Domain\Product\UpdateOptionsFeatureContext
+            - Tests\Integration\Behaviour\Features\Context\Domain\Product\AddProductFeatureContext
+            - Tests\Integration\Behaviour\Features\Transform\SharedStorageTransformContext
+            - Tests\Integration\Behaviour\Features\Transform\StringToBoolTransformContext
+            - Tests\Integration\Behaviour\Features\Context\CountryFeatureContext
+            - Tests\Integration\Behaviour\Features\Context\Domain\CustomerFeatureContext
+            - Tests\Integration\Behaviour\Features\Context\LanguageFeatureContext
+            - Tests\Integration\Behaviour\Features\Context\Domain\FeatureFeatureContext
+            - Tests\Integration\Behaviour\Features\Context\Domain\FeatureValueFeatureContext
+            - Tests\Integration\Behaviour\Features\Context\Domain\Product\UpdateProductFeatureValuesFeatureContext
+            - Tests\Integration\Behaviour\Features\Context\Domain\Product\Combination\GenerateCombinationFeatureContext
+            - Tests\Integration\Behaviour\Features\Context\Domain\AttributeFeatureContext
+            - Tests\Integration\Behaviour\Features\Context\Domain\AttributeGroupFeatureContext
+
+
+
```

Comment:
remove all those empty lines

Reply:
> Nothing critical, just some missing docblocks, so I approve heavy_check_mark

Altough I just noticed Cs fixer is failing, so you could probably fix all those comments too.